tests/unit/test_global_config_validator.py

```diff
@@ -197,6 +197,46 @@ def test_config_validation_when_valid(filename):
             pytest.raises(GlobalConfigValidatorException),
             "Tab 'account' should have entity with field 'name'",
         ),
+        (
+            "invalid_config_configuration_autoCompleteFields_duplicates.json",
+            pytest.raises(GlobalConfigValidatorException),
+            "Duplicates found for autoCompleteFields: Duplicate",
```

Comment:
I think it will look better if we separate the name of the label/entity with single quotes like it is done in the previous examples.

Reply:
good idea, thanks
done